src/main/java/com/worldwarofants/game/module/startmenu/StartMenuController.java

```diff
@@ -0,0 +1,25 @@
+package com.worldwarofants.game.module.startmenu;
+
+import com.worldwarofants.game.arch.AbstractController;
+import com.worldwarofants.game.arch.module.IModuleNavigator;
+import com.worldwarofants.game.module.ModuleName;
+
+public class StartMenuController extends AbstractController<StartMenuView> {
+
+	public StartMenuController(StartMenuView view, IModuleNavigator navigator) {
+		super(view, navigator);
+	}
+
+	public void showStartMenu() {
+		view.renderStartMenuScreen();
+	}
+
+	public void navigateToNewGameModule() {
+		String[] args = new String[0];
+		moduleNavigator.navigateTo(ModuleName.NEW_GAME, args);
+	}
+
+	public void exitGame(){
+		System.exit(0);
```

Comment:
The World class has a method called gameOver if I remember right. Look into how you can use that. Hint: a service

The reason why we should end it through that, instead of just system.exit is because we might have some operations that we will need to do before closing the program. Those operations will be handled through there, whereas just exiting kind of disregards everything else.

Reply:
I'll remove System.exit(0) for now and do a new branch for quitting the game while I figure out services.

Reply:
Alright, sounds good, leave a TODO comment though, on what is supposed to happen there